<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLCDNumber" name="lcdNumberWhite">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="digitCount">
    <number>4</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumberBlack">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>140</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="digitCount">
    <number>4</number>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonstart">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>540</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/start.png</normaloff>:/image/start.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonQuit">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>540</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/end.png</normaloff>:/image/end.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelWhite">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>61</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="image.qrc">:/image/White.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelBlack">
   <property name="geometry">
    <rect>
     <x>723</x>
     <y>60</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="image.qrc">:/image/Black.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>781</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>36</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>萌新与滑稽的对弈</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>lcdNumberWhite</zorder>
  <zorder>lcdNumberBlack</zorder>
  <zorder>toolButtonQuit</zorder>
  <zorder>labelWhite</zorder>
  <zorder>labelBlack</zorder>
  <zorder>toolButtonstart</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
